src/libraries/System.Text.Encodings.Web/src/System/Text/Encodings/Web/DefaultHtmlEncoder.cs

```diff
@@ -68,22 +68,22 @@ internal override int EncodeUtf8(Rune value, Span<byte> destination)
             {
                 if (value.Value == '<')
                 {
-                    if (!SpanUtility.TryWriteBytes(destination, (byte)'&', (byte)'l', (byte)'t', (byte)';')) { goto OutOfSpace; }
+                    if (!"&lt;"u8.TryCopyTo(destination)) { goto OutOfSpace; }
```

Comment:
This library builds for .NET Framework as well. Did these changes regress performance there? Do we care?

Reply:
I guess it's impossible to avoid regressions on Mono and .NET Framework during "de-unsafization", the only part where it won't happen is the tiny coreclr-specific part of the corelib 😞

`""&lt;"u8.TryCopyTo"` seems to be 5ns slower than the old hack on .NET Framework for me locally. But all the public APIs System.Text.Encodings.Web contains aren't fast anyway (e.g. UrlEncoder.Default.Encode takes ~ms) so hopefully it won't be noticed? I've got an impression that this package is not often used on hot path, that's why I started from it.

cc @jkotas (e.g. we recently introduced a similar regression for Mono in https://github.com/dotnet/runtime/pull/108572#issuecomment-2395192427)